pkg/modules/auth/openid/providers.go

```diff
@@ -180,6 +181,16 @@ func getProviderFromMap(pi map[string]interface{}, key string) (provider *Provid
 			usernameFallback = usernameFallbackTypedValue
 		}
 	}
+
+	var forceUserInfo = false
+	forceUserInfoValue, exists := pi["forceuserinfo"]
+	if exists {
+		forceUserInfoTypedValue, ok := forceUserInfoValue.(bool)
+		if ok {
```

Comment:
Maybe worth logging an error when the value is not a bool?

Reply:
I added unit tests for merging `cl` and `cl2`. Because it's not easy to mock the `provider`, so I extract the code of merging 
 from `getClaims` to a new function `mergeClaims`, then test `mergeClaims`